Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM linuxserver/docker-compose
```

Comment:
Why use the image? Maybe we can use Python image so that we won't need to install Python here